### ISSUE TITLE
Class Schedule feed structure is not accessible

### ISSUE THREAD

**Issue by dajoturner** [ucsc/wcms-design-templates#263]
Len Burns submitted this issue. I'm setting it to Critical because we need an assessment of how big this is to fix - I'm assuming it will be Fix Later, but need Jeff to confirm. Is this a Paul Bauman issue?

On the page at:
http://history-dev.ucsc.edu/courses/class-schedule-view.php
There is a structure the purpose of which is to display results of a schedule search.  A quick look leads me to believe this was formatted with CSS/JavaScript.  How ever it was accomplished, from the perspective of a screen reader, it has no structure.  It is reading sequentially.  From the perspective of a screen reader it begins thus:
“Year
Quarter
Class #
Class ID
Class Title
Days
Times
Location
Instructor
Enrollment
Total
Available
Seats
2014
Fall
21221
HIS 10A - 01
U.S. History To 1877
MWF
02:00PM - 03:10PM
J Baskin Engr 152
Jones,C.A.
48
0”
There is no mechanism by which I can navigate the data since it is not within the structure of a data table.  This is a serious problem.
